Gruntfile.js

```diff
@@ -51,4 +51,77 @@ module.exports = function(grunt) {
       ]);
     }
   });
+
+  grunt.registerTask(
+    'travisPullRequestTitleLinting',
+    'get the pull request title in a travis build', function() {
+      if (process.env.TRAVIS_PULL_REQUEST &&
+          process.env.TRAVIS_PULL_REQUEST !== 'false') {
+        var done = this.async();
+
+        // Install ES6/ES7 polyfills required to use changelog-lint programmatically.
+        if (!Object.values) {
+          require('object.values').shim();
+        }
+        if (!Object.entries) {
+          require('object.entries').shim();
+        }
+
+        var pullRequestURLPath = '/repos/' + process.env.TRAVIS_REPO_SLUG +
+          /pulls/ + process.env.TRAVIS_PULL_REQUEST + '.json';
+
+        grunt.log.writeln(
+          'Retrieving the pull request title from https://api.github.com' +
+          pullRequestURLPath
+        );
+
+        require('https').get({
+          host: 'api.github.com',
+          path: pullRequestURLPath,
+          headers: {
+            'User-Agent': 'mozilla web-ext grunt tasks',
+          },
+        }, function(response) {
+          var body = '';
+          response.on('data', function(data) {
+            body += data;
+          });
+          response.on('error', function(err) {
+            grunt.log.writeln('Failed during pull request title download: ' +
+                              err);
+
+            done(false);
+          });
+          response.on('end', function() {
+            var parsed = JSON.parse(body);
```

Comment:
it would be good to start the promise chain with this wrapped in `new Promise(function(resolve) { ... })` just in case it throws a `SyntaxError`. Otherwise, the grunt task might hang.